build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -111,8 +111,8 @@ spec:
                   type: object
                 type: array
               vpcs:
-                description: VPCs describes VPC info, now it includes lb Subnet info
-                  which are needed for AKO.
+                description: VPCs describes VPC info, now it includes Load Balancer
+                  Subnet info which are needed for AKO.
```

Comment:
You might want to expand what AKO means like `... which are needed for the Avi Kubernetes Operator (AKO)`.

Reply:
Updated, thanks